workloads/virtual-machines/image-upload.md

```diff
@@ -0,0 +1,118 @@
+# Creating Virtual Machines from local images with CDI and virtctl
+
+The [Containerized Data Importer](https://github.com/kubevirt/containerized-data-importer) (CDI) project provides facilities for enabling [Persistent Volume Claims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) (PVCs) to be used as disks for KubeVirt VMs.  The three main CDI use cases are:
+
+- Import a disk image from a URL to a PVC (HTTP/S3)
+- Clone an an existing PVC
+- Upload a local disk image to a PVC
+
+This document deals with the third use case.  So you should have CDI installed in your cluster, a VM disk that you'd like to upload, and virtctl in your path.
+
+## Install CDI
+
+Install the latest CDI release [here](https://github.com/kubevirt/containerized-data-importer/releases) (currently v1.3.0)
+
+```bash
+VERSION=v1.3.0
+kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-controller.yaml
+
+```
+
+### Expose cdi-uploadproxy service
+
+The `cdi-uploadproxy` service must be accessible from outside the cluster.  Here are some ways to do that:
+
+- [NodePort Service](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport)
+- [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+- [Route](https://docs.openshift.com/container-platform/3.9/architecture/networking/routes.html)
+- [kubectl port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) (not recommended for production clusters)
+
+Look [here](https://github.com/kubevirt/containerized-data-importer/blob/master/doc/upload.md) for example manifests.
+
+## Supported image formats
+
+- `.img`
+- `.iso`
+- `.qcow2`
+
+\* compressed `.tar`, `.gz`, and `.xz` versions of above supported as well
```

Comment:
Did you mean for this to be escaped?

Reply:
Yeah, intentional.  Asterisk meant to denote footnote.